Sources/AblyAssetTrackingPublisher/DefaultPublisher.swift

```diff
@@ -901,7 +901,6 @@ extension DefaultPublisher: AblyPublisherServiceDelegate {
 // MARK: ResolutionPolicyMethodsDelegate
 extension DefaultPublisher: DefaultResolutionPolicyMethodsDelegate {
     func resolutionPolicyMethods(refreshWithSender sender: DefaultResolutionPolicyMethods) {
-        resolutionPolicy
```

Comment:
I'm a bit confused by what this line has to do with this PR, but I'm also a bit confused by what this line was doing before?

Reply:
Well 😅 I caught this up by accident, and to be honest, this is some orphan that did nothing here. I don't know why this wasn't caught before.

Reply:
OK - probably would be better as a separate PR (or at least a separate commit) but I think we can live with it 🙂